docs/semgrep-code/triage-remediation.md

```diff
@@ -166,6 +179,83 @@ Ignoring a finding through a comment in GitHub changes the status of the finding
 You can also reopen a finding that was previously ignored. To do so, in step 2. of the preceding procedure, use `/semgrep open`. For `/semgrep open` the reason field is optional.
 :::
 
+</TabItem>
+
+<TabItem value='gh'>
+
+:::caution
+This feature is currently in private beta.
+:::
+
+## Prerequisites
+- A **private** GitHub.com repository. This feature is not enabled for public GitHub.com repositories or GitHub Enterprise public and private repositories.
+- You have completed a [Semgrep core deployment](/deployment/core-deployment).
+
+To enable triage through comments:
```

Comment:
I think this needs a header (`## SOME_HEADER`) because otherwise it looks like the process is under **Prerequisites**.